codeworld-api/src/CodeWorld/Driver.hs

```diff
@@ -449,14 +456,91 @@ picToObj' pic =
             ptsJS <- pointsToArr pts
             setProps [("points", ptsJS), ("smooth", pToJSVal True)] obj
             retVal obj
-        Path cs pts w closed smooth -> do
-            obj <- init "path"
+        Polygon cs pts -> do
+            obj <- init "polygon"
             ptsJS <- pointsToArr pts
             setProps
                 [ ("points", ptsJS)
                 , ("width", pToJSVal w)
```

Comment:
Did you mean 0?  You only have a `w` in scope for the thick variants?

Reply:
I tried to pass w for the thick and 0 for the non-thick variants

Reply:
This is Polygon, which is non-thick.  But you have a w here.  I suspect it won't build, since there is no w in scope.

Reply:
Yes. That's what I was struggling to resolve. I tried putting a literal, but it won't accept literals. The Path found its way back in the constructor because I Ctrl+Z-ed a bit. I've removed it.

Reply:
Okay. Got it to type-check.